pkg/audit/manager.go

```diff
@@ -43,7 +45,8 @@ const (
 	msgSize                          = 256
 	defaultAuditInterval             = 60
 	defaultConstraintViolationsLimit = 20
-	defaultListLimit                 = 0
+	defaultListLimit                 = 500
+	defaultOutputDir                 = "/tmp"
```

Comment:
nit: we should have a subdir like `/tmp/audit` in case other tmp files get created in the future